### ISSUE TITLE
support for writeCharacteristic with Priority

### ISSUE THREAD

**Issue by HunanDj** [Polidea/RxAndroidBle#724]
Hello, can you support sending data according to priority? Sometimes for some tasks that have been added to the queue, because the amount of data is relatively large and not important, we hope that it can be sent with a lower priority. Maybe we can have such an interface:
Single<byte[]> writeCharacteristic(@NonNull UUID characteristicUuid, @NonNull byte[] data,Priority priority);

Or can you provide other suggestions, I noticed that there are custom solutions
<T> Observable<T> queue(@NonNull RxBleCustomOperation<T> operation);
But it is too difficult for me.

**Comment by okbean** [comment]
yes strongly recommend  supporting priority queue , suggest that ,there is an incoming call , the app should tell the device  as soon as possible!

**Comment by maras93** [comment]
+1
